infra/cifuzz/continuous_integration.py

```diff
@@ -191,6 +196,23 @@ def build_external_project_docker_image(project_name, project_src,
   return helper.docker_build(command)
 
 
+class ExternalGeneric(BaseCi):
+  def get_diff_base(self):
```

Comment:
not used anywhere?

Reply:
It is used by the parent class in `get_changed_code_under_test`. This class inherits that function.